packages/optimizer/lib/transformers/PreloadHeroImage.js

```diff
@@ -320,8 +322,19 @@ class PreloadHeroImage {
         imgNode.attribs[attr] = node.attribs[attr];
       }
     }
-    node.attribs['i-amphtml-ssr'] = '';
-    node.attribs['data-hero'] = '';
+    // Generate styles for object-fit and object-position
+    const styles = [];
+    const objectFit = node.attribs['object-fit'];
+    if (objectFit) {
+      styles.push(`object-fit:${objectFit}`);
+    }
+    const objectPosition = node.attribs['object-position'];
+    if (objectPosition) {
+      styles.push(`object-position:${objectPosition}`);
+    }
+    if (styles.length > 0) {
+      imgNode.attribs.styles = styles.join(';');
```

Comment:
Don't you mean `style`?

```suggestion
      imgNode.attribs.style = styles.join(';');
```

Reply:
🤦‍♂️